elodie/filesystem.py

```diff
@@ -378,12 +389,22 @@ def process_file(self, _file, destination, media, **kwargs):
         if(move is True):
             stat = os.stat(_file)
             shutil.move(_file, dest_path)
+            if(os.path.exists(_file + '_original')):
```

Comment:
Can we store `_file_original = _file + '_original'` and use that in the following lines as well?